setup.py

```diff
@@ -21,15 +21,22 @@
 ]
 
 install_requires=[
-    'stanford-stk @ git+https://github.com/stanford-futuredata/stk.git@main',
-    'grouped_gemm @ git+https://github.com/tgale96/grouped_gemm@main',
-    'mosaicml-turbo==0.0.4',
+    "triton==2.1.0",
+    "stanford-stk>=0.0.6",
 ]
 
 extra_deps = {}
 
-extra_deps['dev'] = [
-    'absl-py',
+extra_deps["gg"] = [
+    "grouped_gemm @ git+https://github.com/tgale96/grouped_gemm@main",
+]
+
+extra_deps["quant"] = [
+    "mosaicml-turbo==0.0.4",
```

Comment:
```suggestion
    "mosaicml-turbo==0.0.5",
```
@dblalock  can we bump to 0.0.5?

Reply:
Lets do this in a separate PR.

Reply:
+1, I'd rather not touch anything right now